src/test/java/com/yelp/nrtsearch/server/grpc/LuceneServerTest.java

```diff
@@ -277,6 +277,41 @@ public void testStartShard() throws IOException {
     assertTrue(!reply.getSegments().isEmpty());
   }
 
+  @Test
+  public void testStartIndexWithEmptyString() {
+    LuceneServerGrpc.LuceneServerBlockingStub blockingStub = grpcServer.getBlockingStub();
+    try {
+      // start the index without creating any index
+      String emptyTestIndex = "";
+      blockingStub.startIndex(StartIndexRequest.newBuilder().setIndexName(emptyTestIndex).build());
+      fail("The above line must throw an exception");
+    } catch (StatusRuntimeException e) {
+      assertEquals(
+          String.format(
+              "INVALID_ARGUMENT: error while trying to start index since indexName was empty: "),
+          e.getMessage());
+    }
+  }
+
+  @Test
+  public void testCreateAndStartIndexWithEmptyString() {
```

Comment:
Hi @xinyiyelp you can remove this test then and only keep the startIndex test. You can then re-request review for approval.